MobileWallet/Common/UserFeedback/UserFeedback.swift

```diff
@@ -153,8 +153,8 @@ class UserFeedback {
             containerView: containerView,
             image: Theme.shared.images.storeModal!,
             imageTop: imageTop,
-            title: NSLocalizedString("Spend testnet Tari on real (exclusive) stuff", comment: "Store modal"),
-            boldedTitle: NSLocalizedString("", comment: "Store modal"),
+            title: NSLocalizedString("Spend testnet Tari on exclusive stuff", comment: "Store modal"),
+            boldedTitle: NSLocalizedString("exclusive stuff", comment: "Store modal"),
```

Comment:
Should this be `(exclusive) stuff`? @Jasonvdb

Reply:
I usually enforce line breaks too so it matches with the design. Here it would be `Spend testnet Tari on\n...`.

Reply:
Tal wanted it on one line here https://github.com/tari-project/wallet-ios/issues/270#issuecomment-616792741

Reply:
On iPhone 8 when it cannot be on one line it seems to work out
![Simulator Screen Shot - iPhone 8 - 2020-04-21 at 11 16 12](https://user-images.githubusercontent.com/5300488/79851154-389f9580-83c5-11ea-9a74-4fa94d3b9677.png)

Reply:
Ok this copy is out of sync with Android, I shall update.